src/reduction.jl

```diff
@@ -49,8 +36,12 @@ function reduction_op(f, j=f, o...)
                 y=ccall(($F20,$libknet8),$T,(Cint,Ptr{$T}),length(x),x) # do not use @knet8, return not Void
                 @gs; return y
             end
+            
+            $J(x::KnetArray{$T}, d::Int) = $J(x, (d,))
```

Comment:
Oh no, I see you are standardizing the dims argument here.  Was there a problem with the original implementation?  Or did you decide to change following to convention in Base?

Reply:
I cannot remember why I add this. The main drive was to be coherent with Base. I'll look into it